Formula/portable-openssl.rb

```diff
@@ -39,6 +39,7 @@ def arch_args
       {
         :x86_64 => %w[linux-x86_64],
         :i386  => %w[linux-generic32],
+        :arm => %w[linux-armv4],
```

Comment:
Ok, that should be much a problem considering we pass `-march=armv6` flag for compiling.
However since ARMv7 is the oldest docker image, I wouldn't be able to test it for ARMv6. 

@sjackman If you have access to any Raspberry Pi hardware, it would be great if you can help for testing the building results on the real device.